### DISCUSSION TITLE
/admin/api/v1/eventstreams/{collectionName}/views has response 200 and 201

### DISCUSSION THREAD

**Discussion by xdxxxdx** [Informatievlaanderen/VSDS-LDESServer4J#807] [Q&A] [answered]
/admin/api/v1/eventstreams/{collectionName}/views has response 200 and 201.
Just wondering when it should return 200?
As 201 means created. 

![image](https://github.com/Informatievlaanderen/VSDS-LDESServer4J/assets/15314807/4324fe8c-384c-4cdc-8596-35d8da4b33de)

**Comment by Tomvbe** [comment, marked as answer]
Hi @xdxxxdx, thank you for noticing, this is a small bug in the documentation. This should indeed only be 201. I created a PR (https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/808) to fix this.
